AppController/djinn.rb

```diff
@@ -1260,22 +1250,22 @@ def get_app_upload_status(reservation_id, secret)
   #   secret: A string with the shared key for authentication.
   # Returns:
   #   A JSON string with the statistics of the nodes.
-  def get_stats_json(secret)
+  def get_cluster_stats_json(secret)
```

Comment:
I personally prefer the name get_cluster_stats (without _json) since this is the only one to collect stats (that is, there is no the non-json form).